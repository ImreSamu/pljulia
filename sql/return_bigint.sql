CREATE FUNCTION julia_increment(x INTEGER)
RETURNS BIGINT AS $$
    x + 1
$$ LANGUAGE pljulia;
SELECT julia_increment(99999);
DROP FUNCTION julia_increment(x INTEGER);
