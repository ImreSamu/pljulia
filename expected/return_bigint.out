CREATE FUNCTION julia_increment(x INTEGER)
RETURNS BIGINT AS $$
    x + 1
$$ LANGUAGE pljulia;
SELECT julia_increment(99999);
 julia_increment 
-----------------
          100000
(1 row)

DROP FUNCTION julia_increment(x INTEGER);
